pipewelder/cli.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+The Pipelayer command-line interface.
```

Comment:
s/Pipelayer/Pipewelder? I'll stop saying this, but you might want to do a case-insensitive recursive grep for the old project name.